Zend/zend_exceptions.stub.php

```diff
@@ -22,6 +22,13 @@ public function getTraceAsString(): string;
 
 class Exception implements Throwable
 {
+    /** @known */
```

Comment:
I noticed that a few properties are declared by using `ZSTR_KNOWN()` so this phpdoc annotation makes this possible.